src/command_modules/azure-cli-role/azure/cli/command_modules/role/_help.py

```diff
@@ -61,6 +61,8 @@
 helps['ad sp credential'] = """
     type: group
     short-summary: manage a service principal's credentials.
+    long-summary: the credential update will be applied on the Application object the service principal is associated with.
+       In other words, you can accomplish the same thing using "az ad app credential"
```

Comment:
I'm not sure we need to say this... but why do we have a command in two places that does the same thing? We have instances of this elsewhere but I'm curious--if this is the better UX should be deprecate or hide the other one?